controller/hello/hello.go

```diff
@@ -0,0 +1,15 @@
+package hello
+
+import (
+	"fmt"
+	"net/http"
+	"bootcamp/usecase/hello"
+)
+
+/*
+HelloWorld prints a hellow world
+*/
+func HelloWorld(w http.ResponseWriter, r *http.Request) {
```

Comment:
It's ok to have an Index function that handles the `/` route, but the implementation seems a bit over-engineered for what it's actually doing(just printing a string).

I'm guessing you're contemplating "adding" more functionality for the future, that's good though.

Reply:
* This was only my test endpoint :D